docs/Contributing.md

```diff
@@ -128,6 +128,26 @@ useful as a comment when reading the code? Or does it only distract?
 Is the explanation independent of the current context and is it likely
 to stand the test of time?
 
+## Write useful tests
+
+New code requires new tests to be written that will exercise that code.  This
+applies to bug fixes as well as new features.  The tests serve two purposes:
+ensure that the new code does not regress if it needs to be changed and to give 
+an example of what the new code does.
```

Comment:
Can we also say something along these lines?
If new code is exercised by an existing test, the test should be updated to ensure that the new behavior is checked, or a new test that focuses solely on the new behavior should be added.

Reply:
And I think it would be work adding something similar to what the [llvm developer policy](https://llvm.org/docs/DeveloperPolicy.html#test-cases) says about keeping tests small:
> Test cases, especially for regressions, should be reduced as much as possible, by bugpoint or manually. It is unacceptable to place an entire failing program into llvm/test as this creates a time-to-test burden on all developers. Please keep them short.